space-invaders.asm

```diff
@@ -20,9 +20,10 @@
 %define ICON_EXPLOSION_BULLET '#'
 %define ICON_WALL '#'
 
-; GAME WINNERS
-%define PLAYER_WIN 0
-%define INVADERS_WIN 1
+; GAME STATES
+%define STILL_PLAYING 0
+%define PLAYER_WIN 1
+%define INVADERS_WIN 2
```

Comment:
As before, I would encode "type" information as a prefix within the names.

```asm
%define GAME_STATE_PLAYING 0
%define GAME_STATE_PLAYER_WINS 1
%define GAME_STATE_INVADERS_WIN 2
```

Reply:
Sorry, I forgot it...

Reply:
Sorry, I forgot it ...